components/SpotInfo/index.js

```diff
@@ -108,39 +124,14 @@ export default function SpotInfo({ spotId }) {
         {spot.informations && spot.informations.length > 0 ? (
           <EntryList>
             {spot.informations.map((entry) => (
-              <EntryCard key={entry._id}>
-                <EntryTextarea>{entry.info}</EntryTextarea>
-                <EntryDeleteButton onClick={() => handleDeleteEntry(entry._id)}>
-                  <StyledIcon as={PiTrash} size={25} />
-                </EntryDeleteButton>
-              </EntryCard>
+              <EditDeleteInfoForm key={entry._id} entry={entry} spotId={spotId} />
```

Comment:
Yes! That's it 👍
But why do you send the 'spotId' as a prop? I think, this information is already included in the 'entry'-prop

Reply:
I'll look at it again but for now I want to get to my modal. THX